plugins/hbase.yaml

```diff
@@ -0,0 +1,135 @@
+# Plugin Info
+version: 0.0.1
+title: Apache HBase
+description: Log parser for Apache HBase
+parameters:
+  enable_master_log:
+    label: Master Logs
+    description: Enable to collect HBase master logs
+    type: bool
+    default: true
+  master_log_path:
+    label: Master Logs Path
+    description: The absolute path to the HBase master logs
+    type: string
+    default: "usr/local/hbase*/logs/hbase*-master-*.log"
```

Comment:
Missing a leading slash on all the paths

Reply:
Nice catch. Apparently this wasn't a problem in fluentd when they made these defaults in the bundle.